webapp/src/components/getConnectedUsers.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+type Props = {
+    label: string;
+    disabled: boolean;
+};
+
+const GettConnectedUsers = ({label}: Props) => {
+    const handleClick = () => {
+        window.location.href = '/plugins/com.mattermost.msteams-sync/connected-users';
+    };
```

Comment:
IMO, we should use an anchor tag with a download attribute instead of this. Explore and see which is better.